index.js

```diff
@@ -730,27 +733,36 @@ var tick = function() {
 				if(!("units" in node)) {
 					node.units = {};
 				}
-				if(nodeWinner == node.owner || node.owner == -1 || nodeWinner === -1) {
-					if(node.owner != -1) {
-						nodeWinner = node.owner;
+				if (node.owner == -3) {
+					for (var owner in node.units) {
+						// convert all units to energy
+						gd.data.energy[owner] = Math.max(gd.data.energy[owner] + node.units[owner], GAMERULES.MAX_ENERGY);
+						node.units[owner] = 0;
+						logins[gd.users[adjustForRemoved(gd, i)]].conn.send("energy:" + gd.data.energy[owner]);
```

Comment:
Actually, it won't ever work correctly.  You're sending it to a user based on the index of the node, which is always 0, since this is YMH.  Therefore, it will always tell user 0 about everyone's energy.